spec/observables/from-spec.ts

```diff
@@ -210,18 +24,6 @@ describe('Observable.from', () => {
     expect(r).to.throw();
   });
 
-  it('should handle object has observable symbol', (done: MochaDone) => {
```

Comment:
Folded into the matrix test below. I can restore it but I think it would just be a dupe.